quickstart/README.md

```diff
@@ -69,10 +69,19 @@ Fuzzing
 
 Fuzz it:
 
-    $ ~/afl-2.45b/afl-fuzz -i inputs -o out ./vulnerable
+    $ ~/afl-2.45b/afl-fuzz -i in -o out ./vulnerable
 
 For comparison you could also test without the provided example inputs, e.g.:
 
     $ mkdir in
     $ echo "my seed" > in/a
     $ ~/afl-2.45b/afl-fuzz -i in -o out ./vulnerable
+
+POC
+====
+
+Without instrumentation (`-n`): (`afl-fuzz -i inputs -o out -n ./vulnerable`)
+![imaing](https://github.com/enovella/afl-training/blob/master/quickstart/pics/afl-pic-vulnerable-dumbf.png)
+
+With instrumentation (`afl-clang-fast`):
+![imaing](https://github.com/enovella/afl-training/blob/master/quickstart/pics/afl-pic-vulnerable.png)
```

Comment:
A screenshot is a nice idea, thanks. I've added one to master that shows the status once all three easily findable crashes have been detected.